astro/src/content/quickstarts/quickstart-python-flask-web.mdx

```diff
@@ -77,7 +82,7 @@ pip install -r requirements.txt
 
 Now create your Flask app, which for now will consist of an environment file named `.env` and a Python file named `server.py`.
 
-Create the environment file with these contents:
+If you're using the included Docker setup for running FusionAuth, create the environment file with the following contents. If you've chosen to run FusionAuth another way, you'll need to get this information from the first time setup wizard or from your application's View dialog.
```

Comment:
You capitalized `First Time Setup Wizard` above, but not here.  Our content guidelines aren't very clear here, and for things like Hosted Backend we are inconsistent, but I vote for capitalization at all occurances and give this thing a Proper Name

Reply:
Thanks, it's actually supposed to be First-time Setup Wizard, I'm remembering now.